.github/workflows/import/poll-logs.js

```diff
@@ -0,0 +1,270 @@
+// node --env-file=.env .github/workflows/import/poll-logs.js (node >= 21)
+import PQueue from 'p-queue';
+import fs from 'fs';
+import importUrl from './index.js';
+import LOCAL_DEBUG_ENTRIES from './LOCAL_DEBUG_ENTRIES.js';
+import { getImsToken } from './daFetch.js';
+
+const { env, exit } = process;
+const {
+  AEM_LIVE_ADMIN_TOKEN,
+  ROLLING_IMPORT_REPO,
+  LOCAL_RUN,
+  ROLLING_IMPORT_SLACK,
+  GITHUB_SERVER_URL,
+  GITHUB_RUN_ID,
+  LAST_RUN_ISO,
+  ROLLING_IMPORT_CLIENT_SECRET,
+  ROLLING_IMPORT_CLIENT_ID,
+  ROLLING_IMPORT_CODE,
+  ROLLING_IMPORT_GRANT_TYPE,
+  ROLLING_IMPORT_IMS_URL,
+  ROLLING_IMPORT_IMPORT_FROM,
+  ROLLING_IMPORT_LIVE_DOMAIN,
+  ROLLING_IMPORT_ENABLE_DEBUG_LOGS,
+  ROLLING_IMPORT_POLL_LOGS_FROM_REPO,
+  USE_LOCAL_DEBUG_ENTRIES
+} = env;
+
+const toOrg = "adobecom";
+const toRepo = process.env.ROLLING_IMPORT_REPO;
+
+if (!LOCAL_RUN)
+  console.log({
+    AEM_LIVE_ADMIN_TOKEN: !!AEM_LIVE_ADMIN_TOKEN,
+    ROLLING_IMPORT_REPO: !!ROLLING_IMPORT_REPO,
+    LOCAL_RUN: !!LOCAL_RUN,
+    ROLLING_IMPORT_SLACK: !!ROLLING_IMPORT_SLACK,
+    GITHUB_SERVER_URL: !!GITHUB_SERVER_URL,
+    GITHUB_RUN_ID: !!GITHUB_RUN_ID,
+    LAST_RUN_ISO: !!LAST_RUN_ISO,
+    ROLLING_IMPORT_CLIENT_SECRET: !!ROLLING_IMPORT_CLIENT_SECRET,
+    ROLLING_IMPORT_CLIENT_ID: !!ROLLING_IMPORT_CLIENT_ID,
+    ROLLING_IMPORT_CODE: !!ROLLING_IMPORT_CODE,
+    ROLLING_IMPORT_GRANT_TYPE: !!ROLLING_IMPORT_GRANT_TYPE,
+    ROLLING_IMPORT_IMS_URL: !!ROLLING_IMPORT_IMS_URL,
+    ROLLING_IMPORT_IMPORT_FROM: !!ROLLING_IMPORT_IMPORT_FROM,
+    ROLLING_IMPORT_LIVE_DOMAIN: !!ROLLING_IMPORT_LIVE_DOMAIN,
+    ROLLING_IMPORT_ENABLE_DEBUG_LOGS: !!ROLLING_IMPORT_ENABLE_DEBUG_LOGS,
+    ROLLING_IMPORT_POLL_LOGS_FROM_REPO: !!ROLLING_IMPORT_POLL_LOGS_FROM_REPO,
+    USE_LOCAL_DEBUG_ENTRIES: !!USE_LOCAL_DEBUG_ENTRIES,
+  });
+
+const queue = new PQueue({ concurrency: 10 });
+const FROM_PARAM = LOCAL_RUN
+  ? getISOSinceXDaysAgo(1)
+  : encodeURIComponent(LAST_RUN_ISO || getISOSinceXDaysAgo(1));
+
+function getISOSinceXDaysAgo(days) {
+  const now = new Date();
+  now.setDate(now.getDate() - days);
+  //now.setHours(now.getHours() - 4);
+  return now.toISOString();
+}
+
+function getWorkflowRunUrl() {
+  if (GITHUB_SERVER_URL && toRepo && GITHUB_RUN_ID) {
+    return `${GITHUB_SERVER_URL}/${toOrg}/${toRepo}/actions/runs/${GITHUB_RUN_ID}`;
+  }
+  return null;
+}
+
+const slackNotification = (text) => {
+  console.log(text);
```

Comment:
Is this still needed? If so, should it be prefixed with a mention that the message is intended for Slack?

Reply:
The console.log will output it to the action-logs, so yes this is intentional so that if one goes into the action - we have better debug logs